packages/ods/src/utils/select.ts

```diff
@@ -6,6 +6,7 @@
 function mergeSelectedItemPlugin({ label }: Record<string, unknown>): void {
   // @ts-ignore "this" is the TomSelect instance but is set as any by the lib
   const self = this; // eslint-disable-line @typescript-eslint/no-this-alias
+  const counterClassName = 'ts-merged-items__count';
```

Comment:
Don't need to add a constant if you use it once

Reply:
still, other ones are defined here, so it is fine to declare all at the same place, even when used only once